Twilio/Http/CurlClient.php

```diff
@@ -22,6 +25,8 @@ public function request($method, $url, $params = array(), $data = array(),
         $options = $this->options($method, $url, $params, $data, $headers,
                                   $user, $password, $timeout);
 
+        $this->lastRequest = $options;
```

Comment:
Can we add a `$this->lastResponse = null;` so that if an exception occurs and is caught lastResponse wont be set to the previous response?